docs/tutorials/nodes-tutorial.ipynb

```diff
@@ -0,0 +1,376 @@
+    "b = pp.widget_node(slider)\n",
+    "\n",
+    "c = pp.node(gaussian_filter)(a, sigma=b)\n",
+    "\n",
+    "fig = pp.figure2d(c)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c671a291-bf40-40f7-9e03-75dbd77997b2",
+   "metadata": {},
+   "source": [
+    "As expected, the smoothing node `c` now has two parent nodes:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "743c2fdb-9782-439e-9e25-e3afe5960ad3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pp.show_graph(a)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "08ce4da8-21ea-4aa4-bedc-4b38f01fdcdd",
+   "metadata": {},
+   "source": [
+    "And we can display the figure and the slider inside the same container:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9d35d552-af80-46f4-82b3-638d85a2b40a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ipw.VBox([slider, fig])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5dd33c3d-1e0b-4989-b0f2-1c1377364312",
+   "metadata": {},
+   "source": [
+    "When something changes in one of the nodes, all the nodes below it in the graph are notified about the change (the children nodes receive a notification, and they, in turn, notify their own children).\n",
+    "It is then up to each view to decide whether they are interested in the notification or not (usually, most views are interested in all notifications from parents).\n",
+    "If they are, they request data from their parent nodes, which in turn request data from their parents, and so on, until the request has reached the top of the graph.\n",
+    "\n",
+    "As a result, when the slider is dragged, the smoothing node `c` gets notified and tells the figure that a change has occured.\n",
+    "The figure tells node `c` that it wants updated data.\n",
+    "Node `c` asks nodes `a` and `b` for their data.\n",
+    "`a` returns the raw data, while `b` returns the integer value for the kernel size.\n",
+    "`c` then simply send the inputs to the `gaussian_filter` function, and forwards the result to the figure.\n",
+    "\n",
+    "## Multiple views\n",
+    "\n",
+    "To go one step further,\n",
+    "we now wish to add a second one-dimensional figure that will display the sum of the two-dimensional data along the vertical dimension.\n",
+    "On this figure, we would like to display both the original (unsmoothed) data, as well as the smoothed data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "686b3cdf-33b5-4e45-85b9-b2266211c0bc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "a = pp.input_node(da)\n",
+    "\n",
+    "slider = ipw.IntSlider(min=1, max=10)\n",
+    "b = pp.widget_node(slider)\n",
+    "\n",
+    "c = pp.node(gaussian_filter)(a, sigma=b)\n",
+    "\n",
+    "fig2d = pp.figure2d(c)\n",
+    "\n",
+    "# Sum the raw data along the vertical dimension\n",
+    "d = pp.node(sc.sum, dim='yy')(a)\n",
```

Comment:
Why did the earlier example use `partial` to assign `sigma` but here, it uses kwargs of `node`? I'd stick to the latter in all cases.

Reply:
Nice catch. Dropping the need for `partial` further up, as `node` does this for you.